src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/ILGeneratorImpl.cs

```diff
@@ -210,7 +220,32 @@ public override void Emit(OpCode opcode, string str)
             _il.Token(tempVal);
         }
 
-        public override void Emit(OpCode opcode, ConstructorInfo con) => throw new NotImplementedException();
+        public override void Emit(OpCode opcode, ConstructorInfo con)
+        {
+            ArgumentNullException.ThrowIfNull(con);
+
+            if (!(opcode.Equals(OpCodes.Call) || opcode.Equals(OpCodes.Callvirt) || opcode.Equals(OpCodes.Newobj)))
+            {
+                throw new ArgumentException(SR.Argument_NotMethodCallOpcode, nameof(opcode));
+            }
+
+            int stackchange = 0;
+            // Push the return value if there is one.
```

Comment:
Comment seems misplaced \ unnecessary since there is no code to check if there is a return value.